components/Conversations/Components/Summary.tsx

```diff
@@ -31,9 +34,39 @@ const Summary: React.FC<SummaryProps> = ({ transcript, onSummaryGenerated, conve
     setGeneratedSummary(existingSummary || null)
   }, [conversationId])
 
+  const buttons = useTranscriptButtons({
+    message: generatedSummary ?? '',
+    // buttonTypes: ['Copy', 'Share', 'Save', 'SendFeedback']
+    buttonTypes: ['Copy'] // TODO: Add other buttons
+  })
```

Comment:
Doesn't seem like this is actually used anywhere. Can delete?

Reply:
yep!